german/the-black-hack.md

```diff
@@ -93,7 +94,7 @@ Wenn Charaktere Rüstung tragen, die bei ihrer Klasse nicht angegeben ist, addie
 
 ## Rettungswürfe
 
-The Black Hack verwendet keine Rettungswürfe und lässt die Spieler stattdessen eine Eigenschaftsprobe würfeln, wenn sie von irgendeinem Zauber, einer Falle oder einem anderen Effekt betroffen werden - dazu untenstehend eine Orientierung. Beachtet auch die Regel aus dem Abschnitt **Mächtige Gegner**.
+The Black Hack verwendet keine Rettungswürfe, stattdessen lässt es die Spieler eine Eigenschaftsprobe würfeln, wenn sie von irgendeinem Zauber, einer Falle oder einem anderen Effekt betroffen werden - dazu untenstehend eine Orientierung. Beachtet auch die Regel aus dem Abschnitt **Mächtige Gegner**.
```

Comment:
Das ist die fehlende Rettungswurfstelle. Ich sehe keinen Vorteil in der Umformulierung...